### ISSUE TITLE
Cannot install or upgrade to Typecs 4 with nodeclipse installed because of too restrictive chrome sdk dependence

### ISSUE THREAD

**Issue by amergey** [axmor/typecs#81]
It seems that typecs conflicts with nodeclipse because of chrome sdk version. My guess is there is no api changes between chrome 0.3.8 and 0.3.9 sdk, so typecs should probably be less restrictive regarding the version [0.3.8, 0.4.0) or something like that

Trying to install typecs failed with:

Cannot complete the install because of a conflicting dependency.
  Software being installed: TypEcs - TypeScript IDE for Eclipse 4.0.0.201508061529 (com.axmor.eclipse.typescript.feature.feature.group 4.0.0.201508061529)
  Software currently installed: Nodeclipse Node.js .Features Set 0.17.0.201409260936 (org.nodeclipse.enide.nodejs.feature.feature.group 0.17.0.201409260936)
  Only one of the following can be installed at once: 
    Nodeclipse ChromeDevTools SDK 0.3.9.201409260936 (org.chromium.sdk.feature.jar 0.3.9.201409260936)
    ChromeDevTools SDK 0.3.8.201210040400 (org.chromium.sdk.feature.jar 0.3.8.201210040400)

**Comment by p-kudrin** [comment]
Yes you can install any of that version (0.3.9 or 0.3.8). So you should remove TypeEsc, then install Nodeclipse and then reinstall TypEsc

**Comment by amergey** [comment]
For some reasons if nodeclipse is already installed, eclipse only accept to install typecs 1.5.1, and typecs 4.0.0 cannot be installed with error given in my initial message

**Comment by amergey** [comment]
I have proposed the pull request #83 about this issue

**Comment by SolomonSun2010** [comment]
same problem

**Comment by ghost** [comment]
I'm hitting this issue as well with the current TypeEcs update site. Is there an Eclipse update site available with the latest TypEcs build that corrects this?
